client/src/components/ProfileListing/ProfileListing.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { searchProfiles } from '../../helpers/APICalls/searchUsers';
+import { SearchProfileApiData } from '../../interface/Profile';
+import { Box, Grid } from '@mui/material';
+
+import NotFound from '../../pages/NotFound/NotFound';
+import SitterCard from './SitterCard';
+import staticData from './constants';
+
+interface Props {
+  debouncedLocation: string | null;
+  date: string;
+}
+
+const ProfileListing = ({ debouncedLocation, date }: Props): JSX.Element => {
+  const [profiles, setProfiles] = React.useState<SearchProfileApiData | null>(null);
+
+  useEffect(() => {
+    (async () => {
+      if (debouncedLocation && date) {
+        const data = await searchProfiles({ location: debouncedLocation, availability: date });
+
+        setProfiles(data);
+      }
+    })();
+  }, [debouncedLocation, date]);
+
+  const renderProfiles = () => {
+    if (debouncedLocation === 'DEMO') {
+      return staticData.map((sitter, id) => {
+        return (
+          <SitterCard
+            key={id}
+            photo={sitter.imageUrl}
+            name={sitter.maintitle}
+            subTitle={sitter.title}
+            description={sitter.description}
+            address={sitter.country}
+            rate={sitter.price}
+          />
+        );
+      });
+    }
+    if (profiles?.users?.length === 0) {
+      return <NotFound message={'No sitters available'} />;
+    }
+    return profiles?.users?.map((profile, id) => {
+      const { photo, name, description, address } = profile;
+      return (
+        <SitterCard
+          key={id}
+          photo={photo}
+          name={name}
+          subTitle={'Pet Lover'}
+          description={description}
+          address={address}
+          rate={'20/hr'}
```

Comment:
Why is this hard coded

Reply:
In my recent commit, a fallback was added in case there is no rate. The Profile interface doesn't have a `subTitle` property like how the demo data does. I suppose this can be changed when the various systems begin to be integrated.

Reply:
Or thanks for clarification